README.md

```diff
@@ -68,6 +68,11 @@ Python packages with our recommended tooling set up and ready to go.
 
 ## Using this template
 
+Some quick instructions for using our template are below.
+We also have detailed [tutorial](tutorial.md) that has been given in a couple of workshops geared towards researchers at UCL.
```

Comment:
```suggestion
We also have a detailed [tutorial](tutorial.md) that has been given in a couple of workshops geared towards researchers at UCL.
```
You gotta watch that Cunliffe guy. Coming over all bold and adding typos.

Reply:
I've heard terrible things 😉